<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="serialGroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 171, 171);

border-color: rgb(85, 0, 0);</string>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;串  口&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;波特率&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;校验位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;数据位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;停止位&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>151</width>
       <height>201</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="serialCombo">
        <property name="maximumSize">
         <size>
          <width>149</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <item>
         <property name="text">
          <string>串口1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>串口2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>串口3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="boundCombo">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14400</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28800</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="textBitCombo">
        <item>
         <property name="text">
          <string>EVEN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ODD</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NONE</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="dataBitCombo">
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8.5</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stopBitCombo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>绘制曲线</string>
    </property>
    <widget class="QPushButton" name="gorguEBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>入口压力测量功率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="TorqueEBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转矩-效率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="speedEBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转速-测量功率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="liqidEBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>流量-效率</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SpeedTorqueBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转速-转矩</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SpeedRatioBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>转速-效率</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>140</y>
      <width>261</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>历史记录设置</string>
    </property>
    <widget class="QPushButton" name="clearDataBtn">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清除所有数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addItemBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>添加当前行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteLineBtn">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除当前行</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveLocalBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存到本地文件</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openLocalBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开本地文件</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>即时数据</string>
    </property>
    <widget class="QTableWidget" name="data2tableWidget">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>151</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <row>
      <property name="text">
       <string>转速</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>测量功率</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>水力功率</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>效率</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>参数</string>
      </property>
     </column>
    </widget>
    <widget class="QTableWidget" name="data1tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>270</y>
      <width>561</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>历史数据</string>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>311</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(248, 245, 247);</string>
     </property>
     <column>
      <property name="text">
       <string>旁通</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>流量(m3/s)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>入口压力(KPa)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出口压力(KPa)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>转矩(N.m)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>转速(r/min)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>测量功率(w)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>水力功率(w)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>效率</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QLabel" name="curveDisNameLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;拟合曲线&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QFrame" name="widgetOnFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>401</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(184, 254, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>371</width>
       <height>271</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(85, 0, 0);
background-color: rgb(201, 255, 252);</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>81</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 0, 0);
</string>
    </property>
    <property name="title">
     <string>串口开关</string>
    </property>
    <widget class="QPushButton" name="searchSerialBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>搜索串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="switchBtn">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>serialGroupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>groupBox_5</zorder>
   <zorder>widgetOnFrame</zorder>
   <zorder>curveDisNameLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
